website/content/commands/keyring.mdx

```diff
@@ -7,6 +7,8 @@ page_title: 'Commands: Keyring'
 
 Command: `consul keyring`
 
+Corresponding HTTP API Endpoints: [\[VARIES\] /v1/operator/keyring](https://www.consul.io/api-docs/operator/keyring)
```

Comment:
Missing mention of the `keyring:write` ACL required

Reply:
Updated

<img width="946" alt="Screen Shot 2022-01-11 at 9 12 05 AM" src="https://user-images.githubusercontent.com/16387261/148958709-03908aa3-277e-486f-a59e-1b7df7a8a99c.png">